test/parseDetectorTables.test.ts

```diff
@@ -8,18 +8,21 @@ import { ColumnName, detectorTable, findTableByName, getValuesByColumnName, vali
 import { createTestContext, ITestContext } from './global.test';
 
 suite('Detector Dataset Parser', () => {
-    test('Find table by table name', async () => {
+    /* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
Can you do a test-specific eslint file just like we had a test-specifc tslint.json? I would disable these "any" rules for the whole test folder, just like it used to be.